test/integration/ctl/ctl_create_cr_test.go

```diff
@@ -204,18 +229,22 @@ func TestCtlCreateCR(t *testing.T) {
 			}
 
 			// Check the file where the private key is stored
-			expKeyFilename := test.expKeyFilename
-			if test.keyFilename == "" && len(test.inputArgs) == 0 {
-				expKeyFilename = gotCr.Name + ".key"
-			}
-			keyData, err := ioutil.ReadFile(expKeyFilename)
+			keyData, err := ioutil.ReadFile(test.expKeyFilename)
 			if err != nil {
 				t.Errorf("error when reading file storing private key: %v", err)
 			}
 			_, err = pki.DecodePrivateKeyBytes(keyData)
 			if err != nil {
 				t.Errorf("invalid private key: %v", err)
 			}
+
+			// Check the file where the certificate is stored if applicable
+			if test.fetchCert {
```

Comment:
Are these cleaned after the test?

Reply:
Yes they are, see lines 164 and 165 where a cleanup function that deletes the whole tmp folder is used.